### DISCUSSION TITLE
Tabview anchor arrgument

### DISCUSSION THREAD

**Discussion by ShouryaGoje** [TomSchimansky/CustomTkinter#1963] [Q&A]
The anchor arrgument in the tabview is giving me an error the there is no argument name anchor in tabview, so how do i place towards the left side of the frame
![image](https://github.com/TomSchimansky/CustomTkinter/assets/104123911/9814c103-01ed-4c1f-87c4-0e3ada82c9e0)
![image](https://github.com/TomSchimansky/CustomTkinter/assets/104123911/3950018e-24d7-4ef2-acff-4673f93cabca)

**Comment by DragonOfShuu** [comment]
Can you give more details on what is going on? What does your code look like?
